src/RestClient.cc

```diff
@@ -314,8 +307,7 @@ RestResponse Rest::Request(HttpMethod _method,
   }
   else if (_method == HttpMethod::POST_FORM)
   {
-    formpost = BuildFormPost(_form);
-    curl_easy_setopt(curl, CURLOPT_HTTPPOST, formpost);
+    AddFormPost(multipart, _form);
```

Comment:
Should there be the following line here?
```
curl_easy_setopt(curl, CURLOPT_MIMEPOST, multipart);
```

Reply:
good catch! reviewing the deprecation instruction you are right.